app/api/tax.py

```diff
@@ -57,24 +72,13 @@ def query(self, view_kwargs):
             query_ = query_.join(Event).filter(Event.identifier == event.id)
         return query_
 
-    def before_create_object(self, data, view_kwargs):
-        event = None
-        if view_kwargs.get('event_id'):
-            event = safe_query(self, Event, 'id', view_kwargs['event_id'], 'event_id')
-        elif view_kwargs.get('event_identifier'):
-            event = safe_query(self, Event, 'identifier', view_kwargs['event_identifier'], 'event_identifier')
-        if event:
-            data['event_id'] = event.id
-
     view_kwargs = True
-    decorators = (jwt_required, )
```

Comment:
@shubham-padia they need not know the company's tax ID, registered address, etc etc. They'll anyway get who is billing them when they make the payment at paypal or stripe. 

So, just tax rate and whether it is included or not is enough. The rest are all internal details that we add to the invoice we send to the event organizer at the end of each month.